cmd/src/code_intel_upload_flags.go

```diff
@@ -100,9 +100,9 @@ func init() {
 //
 // On success, the global codeintelUploadFlags object will be populated with valid values. An
 // error is returned on failure.
-func parseAndValidateCodeIntelUploadFlags(args []string, isSCIPAvailable bool) (*output.Output, error) {
+func parseAndValidateCodeIntelUploadFlags(args []string) (*output.Output, bool, error) {
 	if err := codeintelUploadFlagSet.Parse(args); err != nil {
-		return nil, err
+		return nil, false, err
```

Comment:
Non-blocking: we default to "scip is not available", but it might make sense to flip the boolean and rename it into `isLSIFRequired` or something like that